src/module/sdk/core/assets/ckb.service.ts

```diff
@@ -39,8 +40,13 @@ export class CKBService {
         to: string,
         amount: bigint,
         privateKeys: string[],
+        sendAllFunds = false,
         feeRate: FeeRate = FeeRate.NORMAL,
     ): Promise<string> {
+        if (sendAllFunds) {
+            // Get total capacity from all non type cells
+            amount = cells.filter((cell) => !cell.cellOutput.type).reduce((ant, act) => ant + BigInt(act.cellOutput.capacity), BigInt(0));
```

Comment:
Mola fent-ho functional xo igual amb un for es mes eficient. Aixi podries filtrar i sumar en cada iteracio. Tampoc nse quantes cells sol haver-hi i si realment tindria algun impacte..

Reply:
O(n) = 2O(n) = O(2n). A part la segon iteració és més curta. És igual de llarg inclús per n grans

Reply:
Les cells que tenen type és perquè són un nft o token o qualsevol cosa amb codi

Reply:
Okey lo del type. I lo altre okey then